faithcomesbyhearing/dbp-web
# FCBHDBP-560 and FCBHDBP-566. The issue with the hls video loading com…

# Description
The issue with the hls video loading component has been fixed, and a new feature has been added to enable searching for the autonym column in the language search functionality.


## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-560
https://fullstacklabs.atlassian.net/browse/FCBHDBP-566

## How Do I QA This
1. Navigate to [live.Bible.is-newdata](http://bibleiswebnodejs18-env.eba-rik39mj2.us-west-2.elasticbeanstalk.com/)
2. Click on bible version tab
3. Clikc on Language tab
4. Input the searching text "bunon" and you should see the result = Bunong - ( Mnong, Central ).
5. Input the searching text "bulg" and you should see the result = Bulgarian.

## Screenshots
- First stable version
[Screencast from 26-04-23 11:00:20.webm](https://user-images.githubusercontent.com/73488660/234633930-b8fe78c3-dd9e-4a46-8296-fda402cee87b.webm)
